services/QuillLMS/lib/tasks/nces_sync.rake

```diff
@@ -36,7 +36,7 @@ namespace :nces_sync do
           grade_range: "#{row['Lowest Grade Offered']} - #{row['Highest Grade Offered']}"
         }
 
-        district = District.where("lower(name) = ?", attributes_hash[:name].downcase).first
+        district = District.where("nces_id = ?", attributes_hash[:nces_id]).first
```

Comment:
Probably worth getting Cissy to confirm this, but my assumption is that the reason we even went for `name` in the first place was because some records in the NCES data file didn't have NCES ID values.  I wonder if what we really want here is "match on nces_id if present, otherwise match on name"?

Reply:
Yeah, good question. I'll ping Cissy since this isn't urgent.

Reply:
Hey y'all -- the reason we were matching on Name and not NCES ID originally is because our district database at the time was very simple. It only had a name field, not an NCES ID field. So we had no choice but to match on name. IIRC the NCES database is pretty complete for districts -- I don't recall any district not having an NCES ID. So I think it would be safe to match purely on NCES ID.

Reply:
Thank you!